ui/components/ui/chip/chip.js

```diff
@@ -14,7 +14,10 @@ export default function Chip({
   leftIcon,
   rightIcon,
   onClick,
+  isInput,
```

Comment:
I think you'll likely want the value to be accessible outside of this chip, so might I suggest:

```suggestion
inputValue,
setInputValue,
```